### PR TITLE
BugFixes

### PR DESCRIPTION
**Pull request by cs-sujeet-ahirrao**
- 1009856 | Use the JSON to grid widget to display the Closed issue on the Apply latest changes record
- 1010258 | Enhancement
    - Removed UUID creation and replace it by issue id in "Create a Request (Issue)" and "Fetch Open Requests (Issues)"
    - Added "Get Server URL" playbook (Sample Collection) iri to fetch the server url in "Setup CICD Environment" playbook
    - Get Server URL and Push Server URL in cicd_env global variable in "Setup Development Environment" and "Setup Production Environment" playbook
    - If user want to Map Source Control Username for different username without Source Control connector configuration then show appropriate comment and error message using "Set Source Control Username" playbook

### CODE REVIEW

File: playbooks/02 - Use Case - CICD/Pull Latest Content from Source Control.json [hunk 1]
```diff
@@ -1,5 +1,4 @@
 {
-    "@context": "\/api\/3\/contexts\/Workflow",
```
Comment (cs-sameer-pathan) [inline playbooks/02 - Use Case - CICD/Pull Latest Content from Source Control.json:2]: Please add this line

File: playbooks/02 - Use Case - CICD/Set Source Control Username.json [hunk 2]
```diff
@@ -577,15 +643,35 @@
             "group": null,
             "uuid": "44d8c9f4-e1dd-4cb3-9925-84ea1a682c54"
         },
+        {
+            "@type": "WorkflowRoute",
+            "name": "Copy of Find Previous Result -> Copy of Update Record",
```
Comment (cs-sameer-pathan) [inline playbooks/02 - Use Case - CICD/Set Source Control Username.json:648]: Route needs to be updated

**Review by cs-sameer-pathan — CHANGES_REQUESTED** [review]

File: playbooks/02 - Use Case - CICD/Pull Latest Content from Source Control.json (hunk 1 above)
Comment (cs-sujeet-ahirrao) [reply playbooks/02 - Use Case - CICD/Pull Latest Content from Source Control.json:2]: will not effect on playbook .. it is removed from platform side

File: playbooks/02 - Use Case - CICD/Set Source Control Username.json (hunk 2 above)
Comment (cs-sujeet-ahirrao) [reply playbooks/02 - Use Case - CICD/Set Source Control Username.json:648]: this will not affect on playbook execution

**Review by cs-sameer-pathan — APPROVED** [review]

**Review by cs-sameer-pathan — APPROVED** [review]

**Review by cs-suyog-jinturkar — APPROVED** [review]

**Pull request merged**